index_votes.py

```diff
@@ -0,0 +1,118 @@
+import logging
+import os
+import textwrap
+from psycopg2.errors import ForeignKeyViolation
+import requests
+from utils import PollingProcess, events_to_process
+
+logger = logging.getLogger(__name__)
+
+
+def fetch_votes_by_proposal(height, proposal_id):
+    resp = requests.get(
+        f"{os.environ['REGEN_API']}/cosmos/group/v1/votes_by_proposal/{proposal_id}",
+        headers={"x-cosmos-block-height": str(height)},
```

Comment:
note: we will hit this issue in redwood:
https://github.com/regen-network/indexer/issues/37